corehq/apps/products/forms.py

```diff
@@ -42,9 +49,16 @@ def __init__(self, product, *args, **kwargs):
         # make sure to select default program if
         # this is a new product
         if not product._id:
-            self.initial['program_id'] = Program.default_for_domain(
-                self.product.domain
-            )._id
+            self.initial['program_id'] = Program.default_for_domain(self.product.domain)._id
```

Comment:
@esoergel noticed that this throws an `AttributeError` both on master and this branch, is there a bootstrapping step i might be missing?